src/ggrc/assets/javascripts/components/assessment_attributes.js

```diff
@@ -0,0 +1,103 @@
+/*!
+    Copyright (C) 2016 Google Inc., authors, and contributors <see AUTHORS file>
+    Licensed under http://www.apache.org/licenses/LICENSE-2.0 <see LICENSE file>
+    Created By: ivan@reciprocitylabs.com
+    Maintained By: ivan@reciprocitylabs.com
+*/
+
+(function (can, $) {
+  can.Component.extend({
+    tag: 'assessment-template-attributes',
+    template: '<content></content>',
+    scope: {
+      fields: new can.List()
+    }
+  });
+  can.Component.extend({
+    tag: 'template-filed',
+    template: can.view(GGRC.mustache_path + '/assessment_templates/attribute_field.mustache'),
+    scope: {
+      removeField: function (scope, el, ev) {
+        var fileds = scope.attr('fields');
+        var field = scope.attr('field');
+        var index = _.findIndex(fileds, function (item) {
+          return item.type === field.type && item.title === field.title;
+        });
+
+        ev.preventDefault();
+        fileds.splice(index, 1);
```

Comment:
I'm not sure how field can be non existing, since we are `template-filed` requires existing field and fields reference, but I can add a check.